api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -366,22 +366,12 @@ public ResponseEntity<Profile> syncComplianceTrainingStatus() {
       complianceTrainingService.syncComplianceTrainingStatus();
     } catch (NotFoundException ex) {
       throw ex;
-    } catch (ApiException | org.pmiops.workbench.absorb.ApiException e) {
+    } catch (ApiException e) {
       throw new ServerErrorException(e);
     }
     return getProfileResponse(userProvider.get());
   }
 
-  @Override
```

Comment:
I confirmed that these 2 are not used by the UI, and therefore are safe to remove. nice!